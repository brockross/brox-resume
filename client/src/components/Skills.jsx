import React from 'react';

const Skills = (props) => {
  return (
    <div>
      <h3>Skills</h3>
      <ul>
        <li>Javascript</li>
        <li>react</li>
        <li>Node</li>
      </ul>
    </div>
  )
}

export default Skills;