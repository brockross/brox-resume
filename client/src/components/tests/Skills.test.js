import React from 'react';
import { shallow } from 'enzyme';

import Skills from  '../Skills.jsx';

describe('Skills', () => {
  it('should render correctly', () => {
    const component = shallow(<Skills/>);

    expect(component).toMatchSnapshot();
  });
});