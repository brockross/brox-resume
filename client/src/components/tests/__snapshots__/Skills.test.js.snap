// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills should render correctly 1`] = `
<div>
  <h3>
    Skills
  </h3>
  <ul>
    <li>
      Javascript
    </li>
    <li>
      react
    </li>
    <li>
      Node
    </li>
  </ul>
</div>
`;
